CODEOWNERS

```diff
@@ -114,6 +114,7 @@ homeassistant/components/sensor/miflora.py @danielhiversen @ChristianKuehnel
 homeassistant/components/sensor/min_max.py @fabaff
 homeassistant/components/sensor/moon.py @fabaff
 homeassistant/components/sensor/netdata.py @fabaff
+homeassistant/components/sensor/nmbs.py. @thibmaek
```

Comment:
The `.` after `.py` should not be there :)

Reply:
Good catch.